sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -192,6 +183,23 @@ class EthService(
     }
   }
 
+  /**
+   * Sign calculates an ECDSA signature for: keccack256("\x19Ethereum Signed Message:\n" + len(message) + message). This
+   * gives context to the signed message and prevents signing of transactions.
+   */
+  @RpcMethod("eth_sign")
+  def sign(sender: Address, message: String): String = {
+    val data = Numeric.hexStringToByteArray(message)
+    val prefix = s"\u0019Ethereum Signed Message:\n${data.length}"
```

Comment:
I think, that every network must define its unique prefix to avoid using message signed in network A to be used in network B.
Can you double check this with other networks?

Reply:
Good point, but GETH has this hardcoded. It seems that replaying signed messages is a common attack vector on smart contracts that use them, not just across different networks. So, smart contract developers have to be aware of this and prevent it anyway: https://medium.com/cryptronics/signature-replay-vulnerabilities-in-smart-contracts-3b6f7596df57

Reply:
That's why i assume we need to hard-code our specific value

Reply:
I wonder if that would break some Ethereum tooling, because they all use this exact value.

Reply:
ok, seems other network use the same message prefix.